### ISSUE TITLE
Hyperparameter sweeps via a CLI flag

### ISSUE THREAD

**Issue by 0jg** [Neuromorphic-Spintronics/neural-sde#52]
Are other architectures, activation functions, or loss functions more conducive to learning neural ODEs/SDEs?

Use e.g., Optuna to do the sweep with a flag e.g., --sweep
